diff --git a/keras_to_tensorflow.py b/keras_to_tensorflow.py
index c66cbbd..486b327 100644
--- a/keras_to_tensorflow.py
+++ b/keras_to_tensorflow.py
@@ -16,9 +16,9 @@ from pathlib import Path
 from absl import app
 from absl import flags
 from absl import logging
-import keras
-from keras import backend as K
-from keras.models import model_from_json, model_from_yaml
+from tensorflow import keras
+from tensorflow.keras import backend as K
+from tensorflow.keras.models import model_from_json, model_from_yaml
 
 K.set_learning_phase(0)
 FLAGS = flags.FLAGS
